package/gluon-web-private-wifi/i18n/de.po

```diff
@@ -16,6 +16,15 @@ msgstr "8-63 Zeichen"
 msgid "Enabled"
 msgstr "Aktiviert"
 
+msgid "Disabled"
+msgstr "Deaktiviert"
+
+msgid "Optional"
+msgstr "Optional"
+
+msgid "Required"
+msgstr "Erforderlich"
+
```

Comment:
- `.pot` file should be updated using `contrib/i18n-scan.pl`
- All `.po` files should be updated using `msgmerge`

(see i18n docs)